source/nodejs/adaptivecards/src/controls/constants.ts

```diff
@@ -1,34 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-/**
- * @deprecated Use Constants.keys.tab instead.
- */
-export const KEY_TAB = 9;
-/**
- * @deprecated Use Constants.keys.enter instead.
- */
-export const KEY_ENTER = 13;
-/**
- * @deprecated Use Constants.keys.escape instead.
- */
-export const KEY_ESCAPE = 27;
-/**
- * @deprecated Use Constants.keys.space instead.
- */
-export const KEY_SPACE = 32;
-/**
- * @deprecated Use Constants.keys.up instead.
- */
-export const KEY_UP = 38;
-/**
- * @deprecated Use Constants.keys.down instead.
- */
-export const KEY_DOWN = 40;
-/**
- * @deprecated Use Constants.keys.delete instead.
- */
-export const KEY_DELETE = 46;
-
```

Comment:
breaking change?

Reply:
No. Thiese constants still exist in the adaptivecards-controls library. They didn't need to exist here.

Reply:
To be clearer, the PopupMenu class and everything related to was brought into the adaptivecards library. The controls library keeps its own copy.